src/PointwiseFunctions/AnalyticSolutions/GrMhd/BondiMichel.hpp

```diff
@@ -23,42 +23,146 @@ namespace grmhd {
 namespace Solutions {
 
 /*!
- * \brief Bondi-Michel accretion with superposed magnetic field in Schwarzschild
- * spacetime in Kerr-Schild coordinates.
+ * \brief Bondi-Michel accretion \cite Michel1972 with superposed magnetic field
+ * in Schwarzschild spacetime in Cartesian Kerr-Schild coordinates.
  *
- * An analytic solution to the 3-D GrMhd system. The user specifies the sonic
+ * An analytic solution to the 3-D GRMHD system. The user specifies the sonic
  * radius \f$r_c\f$ and sonic rest mass density \f$\rho_c\f$, which are the
  * radius and rest mass density at the sonic point, the radius at which the
  * fluid's Eulerian velocity as seen by a distant observer overtakes the local
- * sound speed \f$c_{s,c}\f$. With a specified polytropic exponent \f$\Gamma\f$,
+ * sound speed \f$c_{s,c}\f$. With a specified polytropic exponent \f$\gamma\f$,
  * these quantities can be related to the sound speed at infinity
  * \f$c_{s,\inf}\f$ using the following relations:
  *
  * \f{align*}
- * c_{s,c}^2 &= \frac{1}{2r_c - 3} \\
- * c_{s,\inf}^2 &= \Gamma - 1 + (c_{s,c}^2 - \Gamma + 1)\sqrt{1 + 3c_{s,c}^2}
+ * c_{s,c}^2 &= \frac{M}{2r_c - 3M} \\
+ * c_{s,\infty}^2 &= \gamma - 1 + (c_{s,c}^2 - \gamma + 1)\sqrt{1 + 3c_{s,c}^2}
  * \f}
  *
  * In the case of the interstellar medium, the sound
  * speed is \f$\approx 10^{-4}\f$, which results in a sonic radius of
- * \f$\approx 10^8 M\f$ for \f$\Gamma \neq 5/3\f$ (Rezzola and Zanotti, 2013).
- * However, for numerical-testing purposes, it is more common to use a value of
- * \f$\approx 10 M\f$.
+ * \f$\approx 10^8 M\f$ for \f$\gamma \neq 5/3\f$ \cite RezzollaBook.
  *
  * The density is found via root-finding, through the
  * Bernoulli equation. As one approaches the sonic radius, a second root makes
  * an appearance and one must take care to bracket the correct root. This is
- * done by using the upper bound \f$\frac{\dot{M}}{4\pi}\sqrt{\frac{2}{Mr^3}}\f$
+ * done by using the upper bound
+ * \f$\frac{\dot{M}}{4\pi}\sqrt{\frac{2}{Mr^3}}\f$.
  *
- * Additionally specified by the user are the polytropic exponent \f$\Gamma\f$,
- * and the strength parameter of the magnetic field \f$B\f$.
+ * Additionally specified by the user are the polytropic exponent \f$\gamma\f$,
+ * and the strength parameter of the magnetic field \f$B_0\f$.
  * In Kerr-Schild-Cartesian coordinates \f$(x, y, z)\f$, where
```

Comment:
In Cartesian Kerr-Schild coordinates, so it's consistent with the \brief at the beginning.